packages/color-matching-block/src/types.ts

```diff
@@ -2,22 +2,34 @@
 
 import type { AppBridgeBlock } from '@frontify/app-bridge';
 import type { Color } from '@frontify/fondue';
+import { CSSProperties } from 'react';
 
 export type Props = {
     appBridge: AppBridgeBlock;
 };
 
 export type Settings = {
     imageId?: string;
-    toggleIconIds?: string[];
-    toggleDescription?: string[];
     accentColor?: Color;
     prominentColor?: Color;
     complementaryColors?: Color[];
-    blockTitle?: string;
-    description?: string;
     headingTitle?: string;
     headingDescription?: string;
     sampleImagesIds?: string[];
     showComplementaryColor?: boolean;
+    iconDescriptionFirst?: string;
+    iconDescriptionSecond?: string;
+    useCustomLogo?: boolean;
+    useCustomIcons?: boolean;
+    imageTitle?: string;
+    imageDescription?: string;
+};
+
+export enum ColorMatchingType {
+    Render = 'Render',
+    Toolbar = 'Toolbar',
+}
+
+export type IconProps = {
```

Comment:
Is this used somewhere? Maybe you need to add to the Icons in foundation?

Reply:
Oh good catch :D I've used them in other icons. I added them now also to those icons & moved it to the foundation folder